CONTRIBUTING.md

```diff
@@ -0,0 +1,294 @@
+# Contributing to Vesta
+Thanks so much for helping with the development of Vesta! This guide
+will cover the process for helping out with the project and should allow you
+to get started quickly and easily. If you need any help with contributing,
+please e-mail `stc-reservations{at}yale{dot}edu` or mention
```

Comment:
we do actually have a (general) rails team email, so we should probably use that. It's `stc-developers-rails@elilists.yale.edu`. (Not everyone is on it right now, but I'll fix that)